hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -223,9 +223,10 @@ private void updateLastFlushedSequenceIds(ServerName sn, ServerMetrics hsl) {
       if (existingValue == null || (l != HConstants.NO_SEQNUM && l > existingValue)) {
         flushedSequenceIdByRegion.put(encodedRegionName, l);
       } else if (l != HConstants.NO_SEQNUM && l < existingValue) {
-        LOG.warn("RegionServer " + sn + " indicates a last flushed sequence id ("
-            + l + ") that is less than the previous last flushed sequence id ("
-            + existingValue + ") for region " + Bytes.toString(entry.getKey()) + " Ignoring.");
+        LOG.warn("{} passed a last-flushed-sequenceid={} less than " +
+          "previous-last-flushed-sequenceid={}; reverting to new value",
+            sn, l, existingValue, Bytes.toString(entry.getKey()));
+        flushedSequenceIdByRegion.put(encodedRegionName, l);
```

Comment:
Let's not do this in this patch? This should be resolved by HBASE-23157.

Reply:
Ok. Let me put up new patch.